projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/write-query-management-field.component.ts

```diff
@@ -1,27 +1,20 @@
-import { Component, OnInit } from '@angular/core';
-import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { Component, Input, OnInit } from '@angular/core';
+import { FormGroup } from '@angular/forms';
 import { AbstractFieldReadComponent } from '../base-field';
 import { QueryListItem } from './models';
-import { partyMessagesMockData } from './__mocks__';
 
 @Component({
   selector: 'ccd-write-query-management-field',
   templateUrl: './write-query-management-field.component.html',
   styleUrls: ['./write-query-management-field.component.scss']
 })
 export class WriteQueryManagementFieldComponent extends AbstractFieldReadComponent implements OnInit {
-  public queryItem: QueryListItem;
-  public responseFormGroup = new FormGroup({
-    response: new FormControl('', Validators.required),
-    documents: new FormControl([], Validators.required)
-  });
+  @Input() public queryItem: QueryListItem;
+  public formGroup: FormGroup = new FormGroup({});
 
   constructor() {
     super();
   }
 
-  public ngOnInit(): void {
-    this.queryItem = new QueryListItem();
-    Object.assign(this.queryItem, partyMessagesMockData[0].partyMessages[0]);
-  }
+  public ngOnInit() {}
```

Comment:
@MateiHMCTS - please remove this empty function (it will fail pipeline linting) and remember to remove `implements OnInit` + the relevant import

Reply:
There's some issues when I do that. I think it's because of the class we extend

Reply:
Ah - OK